presidio-anonymizer/presidio_anonymizer/entities/engine/operator_config.py

```diff
@@ -0,0 +1,56 @@
+import logging
+from typing import Dict
+
+from presidio_anonymizer.services.validators import validate_parameter_not_empty
+
+
+class OperatorConfig:
+    """Hold the data of the required operator."""
+
+    logger = logging.getLogger("presidio-anonymizer")
+
+    def __init__(
+            self,
+            operator_name: str,
+            params: Dict = None
+    ):
+        """
+        Create an operator config instance.
+
+        :param operator_name: the name of the operator we want to work with
+        :param params: the parameters the operator needs in order to work
+        """
+        self.logger = logging.getLogger("presidio-anonymizer")
```

Comment:
not sure that a data object should have a logger. its also not used in the code.

Reply:
Remvoed.